src/shared/helpers/cors-origin-parser.helper.ts

```diff
@@ -0,0 +1,5 @@
+export function parseCorsOrigins(origins: string): string[] {
```

Comment:
I think you could have just added this parsing logic inside the env.config 

here is an example:

```js
	ALLOWED_ORIGINS: z
		.string()
		.transform((value) => value.split(",").map((origin) => origin.trim().toLowerCase()))
		.pipe(z.array(z.string())),
```

Reply:
good idea, fixed